runtime/flang/alarm3f.c

```diff
@@ -11,12 +11,13 @@
 
 #ifndef WINNT
 #include <signal.h>
+#include <unistd.h>
 #include "ent3f.h"
 
 /*
 extern void (*signal(int, void (*)(int)))(int);
-*/
 extern int alarm();
+*/
```

Comment:
Why did you decide to keep these lines commented out?

Reply:
A habit of following the encountered pattern (namely, I just followed what original PGI code started)